config/crds/tenancy.kcp.io_workspaces.yaml

```diff
@@ -66,6 +66,19 @@ spec:
             default: {}
             description: WorkspaceSpec holds the desired state of the ClusterWorkspace.
             properties:
+              URL:
```

Comment:
Should this be just `url` instead of uppercase? @ncdc

Reply:
I can't remember offhand what the conventions suggest doing. I'd leave any changes to it to a separate PR, if needed. WDYT?

Reply:
Sure thing, was mostly wondering if that'd be something we'd want to do or not, given that we're introducing a new field under spec and removing the older one